.github/workflows/ci-ubuntu.yml

```diff
@@ -79,7 +79,7 @@ jobs:
             echo "AGDA_HTML_DIR=html/experimental" >> $GITHUB_ENV
           else
             # Pick Agda version for master
-            echo "AGDA_COMMIT=tags/v2.6.4" >> $GITHUB_ENV;
+            echo "AGDA_COMMIT=tags/v2.6.4.1" >> $GITHUB_ENV;
```

Comment:
Do we have a philosophy on major/minor versions of Agda for CI? Perhaps the question really for Agda itself: which versions are regarded as 'stable'/'authoritative' for subsequent development?